tools/scripts/map-deps-to-dist.mjs

```diff
@@ -48,21 +48,23 @@ await asyncForEach(packagesDir, async (packageDir) => {
 
   const dependencies = [...new Set(imports.flat())];
 
-  if (dependencies.length) {
-    const packageJsonPath = DIST_DIR + '/' + packageDir + '/package.json';
+  // Add peerDependencies to dependencies
+  const packageJsonPath = DIST_DIR + '/' + packageDir + '/package.json';
 
-    const packageJson = await readJsonFile(packageJsonPath);
+  const packageJson = await readJsonFile(packageJsonPath);
 
-    // await asyncForEach((dependencies), async (dep) => {
-    //     packageJson.dependencies[dep] = packageJson.version;
-    // })
+  // await asyncForEach((dependencies), async (dep) => {
```

Comment:
are we planning to use this code again soon?

Reply:
not sure, defer to @Ansonhkg , but in any case we can do a future PR to clean that up.